app/poros/direction.rb

```diff
@@ -0,0 +1,9 @@
+class Direction
+  attr_reader :distance,
+              :travel_time
+
+  def initialize(data)
+    @distance = data[:route][:distance].round(2)
+    @travel_time = data[:route][:time]/60
```

Comment:
Ideally, no logic in the initialize method. I think you could round in the view  since rounding is mostly presentation logic, and call an instance method to get the travel time in minutes since dividing by 60 is a calculation. I guess you could argue that rounding is a calculation too.. I'd agree with that.. but I think of rounding mostly for formatting/presentation purposes.